src/corewlan.cs

```diff
@@ -544,4 +544,34 @@ interface CWNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopyi
 	interface CWMutableNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopying  
 	{
 	}
+
+	[BaseType (typeof (NSObject))]
```

Comment:
It's not clear (needs a test) if the default `init` is valid. There's a static method to create an instance (from an interface name) and a shared property.

In doubt it's better to add `[DisableDefaultCtor]` because it's not a breaking change to add it later, but a bad default .ctor would be a breaking change to remove (and can confuse developers to overload the static method).